.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ orbs:
 jobs: # a collection of steps
   build-and-test: # runs not using Workflows must have a `build` job as entry point
     docker: # run the steps with Docker
-      - image: cimg/ruby:3.1-browsers # ...with this image as the primary container; this is where all `steps` will run
+      - image: cimg/ruby:3.1.4-browsers # ...with this image as the primary container; this is where all `steps` will run
```

Comment:
Does 3.1.5 work? I wonder if we could also just get away w/ using 3.1 and let us always pull the latest security patch version. You can see the available image tags here: https://circleci.com/developer/images/image/cimg/ruby

Reply:
Ah gotcha that makes sense. I think it's fine to leave it at whatever matches the app's Ruby verison.